packages/pagebuilder/lib/ContentTypes/Products/products.js

```diff
@@ -257,3 +256,38 @@ Products.propTypes = {
 };
 
 export default Products;
+
+export const GET_PRODUCTS_BY_SKU = gql`
```

Comment:
I opted not to apply this pattern to pagebuilder and treated this the same as deprecated components.